src/bodies.cpp

```diff
@@ -660,13 +660,35 @@ bool bodies::ConvexMesh::containsPoint(const Eigen::Vector3d &p, bool verbose) c
   if (bounding_box_.containsPoint(p))
   {
     Eigen::Vector3d ip(i_pose_ * p);
-    ip = mesh_data_->mesh_center_ + (ip - mesh_data_->mesh_center_) * scale_;
+    ip = mesh_data_->mesh_center_ + (ip - mesh_data_->mesh_center_) / scale_;
```

Comment:
I believe this issue demonstrates a bug due to scale being inverted prior to above fix: https://github.com/ros-planning/moveit_ros/issues/537